misc/config_tools/xforms/vm_configurations.c.xsl

```diff
@@ -296,42 +296,27 @@
       <xsl:variable name="type" select="type/text()" />
       <xsl:variable name="vuart_id" select="position()"/>
       <xsl:value-of select="acrn:initializer(concat('vuart[', $vuart_id, ']'), '{', true())" />
-      <xsl:choose>
-        <xsl:when test="$type = 'legacy'">
+        <xsl:if test="$type = 'legacy'">
```

Comment:
You may want to re-indent the code block. The `xsl:if` is at the same level of the previous `xsl:value-of`.